test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -29,16 +29,17 @@
 #ifndef RUNTIME_MONITOR_H
 #define RUNTIME_MONITOR_H
 
+#include <string>
+#include <vector>
+
 extern "C" {
 #include "wiredtiger.h"
 }
 
-#include "util/debug_utils.h"
-#include "util/api_const.h"
-#include "core/component.h"
-#include "core/throttle.h"
-#include "connection_manager.h"
+#include "core/configuration.h"
+#include "workload/database_model.h"
 #include "workload/database_operation.h"
+#include "util/scoped_types.h"
 
 namespace test_harness {
 /* Static statistic get function. */
```

Comment:
This `get_stat` function is an implementation detail and doesn't need to be exposed in the header.

Reply:
Done.